src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonTableViewCell.swift

```diff
@@ -171,33 +188,18 @@ class HealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierProvid
 	}()
 
 	private lazy var qrCodeContainerStackView: UIStackView = {
-		let stackView = UIStackView(arrangedSubviews: [admissionStateStackView, qrCodeView, segmentedControl])
+		let stackView = UIStackView(arrangedSubviews: [maskAdmissionStatesView, qrCodeView, segmentedControl])
 		stackView.axis = .vertical
 		stackView.spacing = 14.0
 
 		return stackView
 	}()
 
-	private lazy var admissionStateStackView: UIStackView = {
-		let stackView = AccessibleStackView(arrangedSubviews: [admissionStateTitleLabel, admissionStateView])
-		stackView.spacing = 8.0
-		stackView.alignment = .center
-
-		return stackView
-	}()
-
-	private let admissionStateTitleLabel: ENALabel = {
-		let admissionStateTitleLabel = ENALabel(style: .headline)
-		admissionStateTitleLabel.numberOfLines = 0
-		admissionStateTitleLabel.textColor = .enaColor(for: .textPrimary1)
-		admissionStateTitleLabel.text = AppStrings.HealthCertificate.Overview.admissionStateTitle
-		admissionStateTitleLabel.setContentCompressionResistancePriority(.init(rawValue: 998), for: .horizontal)
-
-		return admissionStateTitleLabel
-	}()
-
 	private lazy var admissionStateView = RoundedLabeledView()
+	private lazy var maskStateView = RoundedLabeledView()
 
+	private lazy var maskAdmissionStatesView = UIView()
```

Comment:
If this was still a `AccessibleStackView` it would be handled more elegantly for very large font sizes

Reply:
Will be handled in https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13883